charts/otel-cloud-stack/values.yaml

```diff
@@ -737,3 +737,34 @@ logsCollector:
           exporters: [otlp]
 
 collectors: []
+
+opAMPBridge:
```

Comment:
did you check this rendered as expected?

Reply:
yup, it looks alright to me aside from the clusterrole and clusterrolebinding names being different (which may be fine, i'm not sure if those names have specific needs)

Reply:
yeah that doesn't matter as long as they bind the right serviceaccount for the bridge

Reply:
ok yeah looks like its all fine, i was able to spin up an install with the bridge successfully

Reply:
bridge isn't able to list collectors with the way this is right now— here's what names are being set to:
serviceaccount: `otel-cloud-stack-opamp-bridge`
clusterrole: `otel-cloud-stack-bridge`
clusterrolebinding: `otel-cloud-stack`, binds role  `otel-cloud-stack-bridge` to serviceaccount `otel-cloud-stack-bridge-opamp-bridge` (this serviceaccount doesnt exist)